Userland/Utilities/unzip.cpp

```diff
@@ -72,18 +72,54 @@ static bool unpack_zip_member(Archive::ZipMember zip_member, bool quiet)
     return true;
 }
 
+static bool is_extract_match(StringView file_name, StringView filter_text)
```

Comment:
Should we reimplement this here?
I'd suggest translating `_` to `?` and just passing the pattern and file name to String::matches.

Reply:
I switched to your suggestion and it's working just fine! The only thing I found was it seemed to not have matched properly on a double underscore at the end of the filter text, but I consider that extreme edge-casey enough to let it slide,

Reply:
there might of course be bugs in the impl of globs, but this way everyone will get the fixes and/or speedups whenever that function is improved
(looks like consecutive question marks are fine though, try `match foo { f?? { echo yes } }` in the shell)